src/spikeinterface/postprocessing/isi.py

```diff
@@ -256,16 +197,14 @@ def compute_isi_histograms_numba(sorting, window_ms: float = 50.0, bin_ms: float
 if HAVE_NUMBA:
 
     @numba.jit(
-        (numba.int64[:, ::1], numba.int64[::1], numba.int32[::1], numba.int32, numba.int32, numba.float32),
+        (numba.int64[:, ::1], numba.int64[::1], numba.int32[::1], numba.float64[::1]),
         nopython=True,
         nogil=True,
         cache=True,
-        parallel=True,
     )
-    def _compute_isi_histograms_numba(ISIs, spike_trains, spike_clusters, max_time, bin_size, sampling_f):
+    def _compute_isi_histograms_numba(ISIs, spike_trains, spike_clusters, bins):
         n_units = ISIs.shape[0]
 
-        for i in numba.prange(n_units):
+        for i in range(n_units):
```

Comment:
```suggestion
        units_loop = numba.prange(n_units) if n_units > 300 else range(n_units)
        for i in units_loop:
```
I guess that the actual breakeven point will depend both on the number of units and the length of the spike trains, but as a rule of thumb I think that only using the number of units would work. @zm711 dows this make sense?

Reply:
The length of the spike trains are done sequentially regardless so for this optimization we can try to do the units in parallel, but I wouldn't have a clue how to parallelize doing the actually spike train counting. (Long-winded way of saying I agree with you--had to think it out myself :) ). 

To your point though, I haven't actually tested to see where the value of spinning threads actually switches to a net benefit vs a net cost. 300 seems low to me (I would imagine doing 500) but we are both just picking round numbers I think? So 300 sounds works for me!